crates/formality-check/src/impls.rs

```diff
@@ -17,10 +17,8 @@ use formality_types::{
 };
 
 impl super::Check<'_> {
-    #[context("check_trait_impl({v:?})")]
-    pub(super) fn check_trait_impl(&self, v: &TraitImpl) -> Fallible<()> {
-        let TraitImpl { binder, safety } = v;
```

Comment:
JFYI I use struct patterns a lot as a deliberate style choice -- I want compilation errors when/if add'l fields are added to the struct -- but in this case it doesn't seem very critical.

Reply:
It's more for the result when opening `binder`

Reply:
Make sense. I would add this back in this PR but it's already merged :) I'll open another PR.

Reply:
Even if it's not critical per se, I've re-introduced it here for consistency and easier evolvability, in #156 -- as well as to the negative impl counterpart, which didn't have it for instantiating its binder.